tests/test_meshing_queries.py

```diff
@@ -0,0 +1,123 @@
+from ansys.fluent.core import examples
+
+import_filename = examples.download_file(
+    "elbow.cas.h5", "pyfluent/examples/DOE-ML-Mixing-Elbow"
+)
+
+
+def test_meshing_queries(new_mesh_session):
+    meshing_session = new_mesh_session
+    meshing_session.tui.file.read_case(import_filename)
+
+    assert meshing_session.meshing_queries.GetFaceZoneAtLocation(
+        [1.4, 1.4, 1.4]
+    ) == meshing_session.scheme_eval.scheme_eval(
+        "(get-face-zone-at-location '(1.4 1.4 1.4))"
+    )
+
+    assert meshing_session.meshing_queries.GetZonesOfType(
+        "velocity-inlet"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-zones-of-type 'velocity-inlet)")
+
+    assert meshing_session.meshing_queries.GetZonesOfGroup(
+        "inlet"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-zones-of-group 'inlet)")
+
+    assert meshing_session.meshing_queries.GetFaceZonesOfFilter(
+        "*"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-face-zones-of-filter '*)")
+
+    assert meshing_session.meshing_queries.GetCellZonesOfFilter("*") == [3460]
+    assert meshing_session.scheme_eval.scheme_eval("(get-cell-zones-of-filter '*)") == (
+        3460,
+    )
```

Comment:
@seanpearsonuk @mkundu1 We can not compare all results as it is. We are getting correct results but with different return types. Hence we have compared above 2 same results separately.